cclib/parser/data.py

```diff
@@ -169,8 +193,13 @@ class ccData:
        "scfenergies":      Attribute(numpy.ndarray,    'scf energies',                'optimization:scf'),
        "scftargets":       Attribute(numpy.ndarray,    'targets',                     'optimization:scf'),
        "scfvalues":        Attribute(list,             'values',                      'optimization:scf'),
+       "solvation":        Attribute(str,              'solvation',                   'metadata'),
+       "symmno":           Attribute(int,              'symmetry number',             'properties:rotational'),
+       "td_energy":        Attribute(list,             'TD energy',                   'optimization:TD'),
+       "td_energy_fin":    Attribute(float,            'TD energy',                   'properties:energy'),
        "temperature":      Attribute(float,            'temperature',                 'properties'),
        "time":             Attribute(numpy.ndarray,    'time',                        'N/A'),
+       "times_converged":  Attribute(numpy.ndarray,    'times converged',             'optimization'),

```

Comment:
This is a tricky point, and it's designed to address a freq convergence issue from Gaussian (detailed here https://gaussian.com/faq3). In this case, AQME checks the "times_converged" value to figure out whether a calculation converged during a freq calculation. In the current cclib version, this error isn't detected with freqdone/optdone.

Reply:
Ah sorry, I wasn't clear, what I meant was essentially keeping the same logic but renaming `times_converged` to `freqdone` (which doesn't exist currently). The implementation here isn't really counting anyway (because it stops after 2), so I think we might as well make it an explicit yes/no flag